app/assets/javascripts/admin/addon/controllers/admin-site-settings.js

```diff
@@ -22,6 +22,17 @@ export default class AdminSiteSettingsController extends Controller {
     return 100;
   }
 
+  sortSettings(settings) {
+    // Sort the site settings so that fuzzy results are at the bottom
+    // and ordered by their gap count asc.
+    settings.sort((a, b) => {
+      const aSort = a.sort === undefined ? 0 : a.sort;
```

Comment:
I think just having this called `.sort` is a little confusing; something like `.weight` or `.sortWeight` would be better IMO

Reply:
Ah yes, thanks for the name suggestion, always appreciated.